tests/unit/test_metrics.py

```diff
@@ -0,0 +1,46 @@
+"""Testing functions of metrics module."""
+
+from datetime import datetime
+
+import numpy as np
+import polars as pl
+import pytest
+
+from wristpy.core.models import Measurement
+from wristpy.processing.metrics import euclidean_norm_min_one
+
+TEST_LENGTH = 100
+
+
+@pytest.mark.parametrize(
+    "x,y,z, expected_enmo",
+    [
+        (np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 0),
+        (np.ones(TEST_LENGTH), np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 0),
+        (np.ones(TEST_LENGTH) * 2, np.zeros(TEST_LENGTH), np.zeros(TEST_LENGTH), 1),
+        (np.ones(TEST_LENGTH), np.ones(TEST_LENGTH) * 2, np.ones(TEST_LENGTH) * 2, 2),
+    ],
+)
+def test_euclidean_norm_min_one(
+    x: np.ndarray, y: np.ndarray, z: np.ndarray, expected_enmo: float
+) -> None:
+    """Tests the euclidean norm function."""
+    dummy_date = datetime(2024, 5, 2)
+    dummy_datetime = pl.Series("time", [dummy_date] * TEST_LENGTH)
+    test_acceleration = Measurement(
+        measurements=np.column_stack((x, y, z)), time=dummy_datetime
```

Comment:
Actually, this is interesting that Measurement accepts this. All the timestamps are the exact same value. We validate that the time  field is sorted, and apparently this counts. @ReinderVosDeWael should we concerned about this in our Measurement model?

Reply:
Depends; is there ever a case where you expect the watch to return multiple measurements for the same timepoint? If no, then yeah I'd double check that all time values are unique.

Reply:
Definitely should not have the exact same timestamp for multiple data points.

Yes. I can create a new task to add that as part of the time field validator. That being said it will then cause this unit test to fail  on the Measurement() initialization?

Reply:
That's fine - since we're aware of the issue already you could get ahead of it now @frey-perez if you want, but in general old tests failing due to changes in architecture is fine, so long as the tests are updated to reflect that new architecture.  

As far as I'm concerned it can be handled in either PR.